docs/resources/blocks-and-transactions.md

```diff
@@ -0,0 +1,1560 @@
+# Crypto.org Chain Blocks and Transactions
+
+This document describes the block and transaction structure of the Crypto.org Chain and explains different ways to extract and parse the details of them.
+
+## Table of Content
+
+- [Crypto.org Chain Blocks and Transactions](#cryptoorg-chain-blocks-and-transactions)
+  - [Table of Content](#table-of-content)
+  - [Common APIs](#common-apis)
+    - [1. Tendermint Block API](#_1-tendermint-block-api)
+    - [2. Tendermint Block Results API](#_2-tendermint-block-results-api)
+    - [3. Cosmos Transaction Query API](#_3-cosmos-transaction-query-api)
+    - [4. Cosmos Transaction Search API](#_4-cosmos-transaction-search-api)
+  - [Common Block Details](#common-block-details)
+    - [1. Mint](#_1-mint)
+    - [2. Block Rewards](#_2-block-rewards)
+    - [3. Proposer Rewards](#_3-proposer-rewards)
+    - [4. Commissions](#_4-commissions)
+  - [Common Transaction Details](#common-transaction-details)
+    - [1. Block Height](#_1-block-height)
+    - [2. Transaction Hash](#_2-transaction-hash)
+    - [3. Transaction Fee](#_3-transaction-fee)
+    - [4. Assets and Amount](#_4-assets-and-amount)
+      - [1. Single object](#_1-single-object)
+      - [2. Array](#_2-array)
+      - [3. String](#_3-string)
+  - [Bank](#bank)
+    - [1. MsgSend](#_1-msgsend)
+    - [2. MsgMultiSend](#_2-msgmultisend)
+  - [Distribution](#distribution)
+    - [1. MsgSetWithdrawAddress](#_1-msgsetwithdrawaddress)
+    - [2. MsgWithdrawDelegatorReward](#_2-msgwithdrawdelegatorreward)
+    - [3. MsgWithdrawValidatorCommission](#_3-msgwithdrawvalidatorcommission)
+    - [4. MsgFundCommunityPool](#_4-msgfundcommunitypool)
+  - [Staking](#staking)
+    - [1. MsgCreateValidator](#_1-msgcreatevalidator)
+    - [2. MsgEditValidator](#_2-msgeditvalidator)
+    - [3. MsgDelegate](#_3-msgdelegate)
+    - [4. MsgBeginRedelegate](#_4-msgbeginredelegate)
+    - [5. MsgUndelegate](#_5-msgundelegate)
+    - [5b. Upon MsgUndelegate completed](#_5b-upon-msgundelegate-completed)
+  - [Slashing](#slashing)
+    - [1. MsgUnjail](#_1-msgunjail)
+    - [2. Being Jailed and Slashed](#_2-being-jailed-and-slashed)
+      - [Liveness](#liveness)
+      - [Double Sign](#double-sign)
+      - [Limitations](#limitations)
+  - [Governance](#governance)
+    - [1. MsgSubmitProposal](#_1-msgsubmitproposal)
+    - [1b. Community Pool Spend Proposal](#_1b-community-pool-spend-proposal)
+    - [2. MsgDeposit](#_2-msgdeposit)
+    - [2a. Burn Proposal's Deposit if Proposal does not Get Enough Deposit](#_2a-burn-proposals-deposit-if-proposal-does-not-get-enough-deposit)
+    - [2b. Return Proposal's Deposit](#_2b-return-proposals-deposit)
+    - [2c. Burn Proposal's Deposit](#_2c-burn-proposals-deposit)
+    - [3. MsgVote](#_3-msgvote)
+    - [4. Proposal Result](#_4-proposal-result)
+  - [Appendix: Module Accounts on Mainnet](#appendix-module-accounts-on-mainnet)
+
+## Common APIs
+
+### Tendermint
```

Comment:
```suggestion
```

Reply:
I will keep it as the common APIs examples provide are
- Tendermint-type (port `26657`); and
- Cosmos-type  (port `1317`).